circle.yml

```diff
@@ -0,0 +1,17 @@
+machine:
+  xcode:
+    version: 8.2
+
+dependencies:
+  pre:
+    - gem install xcpretty --no-rdoc --no-ri --no-document --quiet
+    - if [[ $CIRCLE_BRANCH == master ]]; then pod setup; fi
+  cache_directories:
+    -  "~/.cocoapods/repos/master"
```

Comment:
@djbe Well I think we still need to change that `pod setup` + `cache_directories` strategy to use what @dantoml told us about in Slack, to improve that part before consider merging.